int16 gui_key
int16 expandfield
int16 refbox_cyan
int16 refbox_magenta
int16 team